src/deployment-guides/resources/tko-on-aws/tkg_vpc/config_files/harbor-data-values.yaml

```diff
@@ -0,0 +1,215 @@
+---
+#! The namespace to install Harbor
+namespace: tanzu-system-registry
+
+#! The FQDN for accessing Harbor admin UI and Registry service.
+hostname: harbor.svc.cluster.local
+#! The network port of the Envoy service in Contour or other Ingress Controller.
+port:
+  https: 443
+
+#! The log level of core, exporter, jobservice, registry. Its value is debug, info, warning, error or fatal.
+logLevel: info
+
+#! [Optional] The certificate for the ingress if you want to use your own TLS certificate.
+#! We will issue the certificate by cert-manager when it's empty.
+tlsCertificate:
+  #! [Required] the certificate
+  tls.crt:
+  #! [Required] the private key
+  tls.key:
+  #! [Optional] the certificate of CA, this enables the download
+  #! link on portal to download the certificate of CA
+  ca.crt:
+
+#! Use contour http proxy instead of the ingress when it's true
+enableContourHttpProxy: true
+
+#! [Required] The initial password of Harbor admin.
+harborAdminPassword: "passwordpassword"
```

Comment:
We need to do something with this before it hits master. Should we randomly generate all of these or all them all as inputs?

Reply:
these are the sample password , user should enter real password and other values. i added sample password and all mandatory fields into to these template files with sample values  , otherwise they will throw validation errors during package creation. instructions given into read me file